setup.cfg

```diff
@@ -57,7 +57,7 @@ install_requires =
     # koji
     cccolutils
     lazy_object_proxy
-    marshmallow>=3.0.0
+    marshmallow>=2.0.0
```

Comment:
Do you think the code will work with v3 as well? If not, then we should probably use [compatible release operator](https://www.python.org/dev/peps/pep-0440/#compatible-release) instead, i.e.
```
marshmallow~=2.0
```
which is equivalent to `>=2.0` and `==2.*`

Reply:
Changed